packages/core/src/codewhispererChat/controllers/chat/telemetryHelper.ts

```diff
@@ -338,6 +338,7 @@ export class CWCTelemetryHelper {
             cwsprChatResponseLength: message.messageLength,
             cwsprChatConversationType: 'Chat',
             credentialStartUrl: AuthUtil.instance.startUrl,
+            codewhispererCustomizationArn: triggerPayload.customization.arn,
```

Comment:
does this need to avoid empty string similar to the logic in converter.ts ?

Reply:
nahh, cuz here it's Toolkit telemetry API where empty string is allowed, but we definitely need the same afterward when we're doing our service sendTelemetryEvent API